src/encoded/search.py

```diff
@@ -930,10 +978,14 @@ def initialize_facets(request, doc_types, prepared_terms, schemas):
     # hold disabled facets from schema; we also want to remove these from the prepared_terms facets
     disabled_facets = []
 
+    current_type_schema = request.registry[TYPES][doc_types[0]].schema
+    initialize_additional_facets(request, doc_types, additional_facets,
+                                 append_facets, current_type_schema)
+
+
     # Add facets from schema if one Item type is defined.
     # Also, conditionally add extra appendable facets if relevant for type from schema.
     if len(doc_types) == 1 and doc_types[0] != 'Item':
```

Comment:
Seems like you've already presumed the doc_types is length 1 above, no? Anyway, if that logic were done as I suggested above could you still use that variable here?